Error on webpack 5

Hello everyone
I installed `@wazo/sdk` latest version (`0.39.0-beta.3`) on `create-react-app` latest version project that use webpack 5 and also I have an older CRA project that I install latest `@wazo/sdk` on that which use webpack 4 but I have many errors about export in first while in latter one all things is OK and I have no problem.
Of course webpack 5 is OK with `@wazo/sdk` version `0.38.x` and has no problem with it.


Error in terminal:
`Module not found: Error: Package path ./lib/simple is not exported from package .../node_modules/@wazo/sdk (see exports field in .../node_modules/@wazo/sdk/package.json)`

Error in browser:
`Uncaught Error: Cannot find module '@wazo/sdk/lib/simple'`


Thank you for your reply soon. 🙏🏼

--- a/README.md
+++ b/README.md
@@ -1449,3 +1449,65 @@ ws.updateToken(newToken: string);
 ```js
 ws.close();
 ```
+
+
+# Usage with webpack 5
+
+As webpack 5 dropped some polyfill, you have to add them:
+
+```sh
+yarn add -D assert buffer https-browserify url stream-http stream-browserify browserify-zlib process
+```
+
+If you have used `create-react-app` to create your app, you can customize the webpack configuration with `react-app-rewired` :
+
+```sh
+yarn add -D react-app-rewired
+```
+
+Create a `config-overrides.js` file :
+```js
+const webpack = require('webpack');
+module.exports = function override(config, env) {
+  config.resolve.fallback = {
+    ...config.resolve.fallback,
+    assert: require.resolve("assert"),
+    buffer: require.resolve('buffer'),
+    https: require.resolve('https-browserify'),
+    url: require.resolve('url'),
+    http: require.resolve('stream-http'),
+    stream: require.resolve('stream-browserify'),
+    zlib: require.resolve('browserify-zlib'),
+  };
+
+  config.plugins.push(
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+      Buffer: ['buffer', 'Buffer'],
+    }),
+  );
+
+  config.module.rules.push({
+    test: /\.m?js/,
+    resolve: {
+      fullySpecified: false
+    }
+  })
+
+  return config;
+}
+```
+
+Then edit in `package.json` :
+```json
+"scripts": {
+  // ...
+  "start": "react-app-rewired start",
+  "build": "react-app-rewired build",
+  "test": "react-app-rewired test",
+  "eject": "react-app-rewired eject"
+  // ...
+},
+```
+
+Please refer to https://github.com/facebook/create-react-app/issues/11756 for more details.

--- a/package.json
+++ b/package.json
@@ -49,7 +49,8 @@
         "types": "./lib/types/index.d.ts",
         "default": "./lib/index.js"
       }
-    }
+    },
+    "./lib/simple": "./lib/simple/index.js"
   },
   "dependencies": {
     "events": "^3.3.0",

